src/main/java/com/faforever/client/fx/JavaFxUtil.java

```diff
@@ -488,4 +490,29 @@ public static void runLater(Runnable runnable) {
   public static void bindManagedToVisible(Node... nodes) {
     Arrays.stream(nodes).forEach(node -> node.managedProperty().bind(node.visibleProperty()));
   }
+
+  public static void bindTextFieldAndRangeSlide(TextField textField, RangeSlider rangeSlider, boolean highValue) {
```

Comment:
Rather than a boolen highValue just pass both the high and lo value text fields
```suggestion
  public static void bindTextFieldAndRangeSlide(TextField lowValueTextField, TextField highValueTextField, RangeSlider rangeSlider) {
```